core/loadpoint_test.go

```diff
@@ -525,6 +528,7 @@ func TestChargedEnergyAtDisconnect(t *testing.T) {
 		chargeMeter: &Null{}, // silence nil panics
 		chargeRater: rater,
 		chargeTimer: &Null{}, // silence nil panics
+		wakeUpTimer: NewTimer(),
```

Comment:
könntest Du den noch in 

    wakeupTimer

umbenennen? Nicht super konsistent, aber dann passts wenigstens zur timer.go